### PR TITLE
Mat 4230 terminology support automated tests

### PR DESCRIPTION
**Pull request by bwelch8105**

### CODE REVIEW

File: cypress/Shared/TestCaseJson.ts [hunk 1]
```diff
@@ -11,6 +11,846 @@ export class TestCaseJson {
...
+  '{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}' +
+  '{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}' +
+  '{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}'+
+  +'{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}'+
+  '{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}'+
+  '{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}'+
+  '{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}'+
+  '{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}'+
+  '{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}'+
+  '{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}{del}'+
```
Comment (cvasile001) [inline cypress/Shared/TestCaseJson.ts:628]: what is happening here? lol

File: cypress/integration/WebInterface/Test Cases/TestCaseValidations.spec.ts [hunk 2]
```diff
@@ -887,3 +897,851 @@ describe('Test Case Run Test Case button validations', () => {
...
+        cy.get(TestCasesPage.runTestButton).click()
+
+        cy.get(TestCasesPage.testCalculationResults).should('contain', 'Population Group: population-group-1')
+
+        cy.get(TestCasesPage.testCalculationResultsLineTwo).should('contain.text', '\ndefine "Qualifying Encounters":\n  (\n    [Encounter: "Office Visit"]\n          union [Encounter: "Annual Wellness Visit"]\n  ) ValidEncounter\n        where ValidEncounter.period during "Measurement Period"\n            and ValidEncounter.status  = \'finished\'\n')
+        cy.get(TestCasesPage.testCalculationResultsLineThree).should('contain.text', '\ndefine "Initial Population":\n  exists "Qualifying Encounters"\n')
+    })
+})
+
+describe.only('Test Case JSON / terminology tests: positive tests -- Test Case JSON uses value set(s)', () => {
```
Comment (cvasile001) [inline cypress/integration/WebInterface/Test Cases/TestCaseValidations.spec.ts:1489]: do we need this .only?

**Review by cvasile001 — CHANGES_REQUESTED** [review]
I think you need to remove the .only in TestCaseValidations

**Review by cvasile001 — APPROVED** [review]
all good

File: cypress/Shared/TestCaseJson.ts (hunk 1 above)
Comment (bwelch8105) [reply cypress/Shared/TestCaseJson.ts:628]: JSON value was / is wonky and required that I type it in the field with all format considerations. This ended up causing a bunch of extra characters and spaces at the end of the value. I just took the approach of deleting them.

File: cypress/integration/WebInterface/Test Cases/TestCaseValidations.spec.ts (hunk 2 above)
Comment (bwelch8105) [reply cypress/integration/WebInterface/Test Cases/TestCaseValidations.spec.ts]: nope
it has been removed

**Pull request merged**